lib/grid.rb

```diff
@@ -17,9 +17,23 @@ def get_value(coordinate)
     structure[coordinate]
   end
 
+  def get_computer_choice
+    get_random_key
```

Comment:
This layer of indirection probably isn't necessary. You could have just one well named method.

Reply:
✔️